sections/3_societal_impact/scientific_literacy.qmd

```diff
@@ -18,30 +18,37 @@ affiliations:
 
-Second, [@laugksch2000] and [@norris2014] conceptualise scientific literacy to comprise of three different interpretations of what it means to be ‘literate’. The first refers to what one has learned – the specific knowledge gained. The second refers to being competent, having a certain capacity to engage with scientific contents. The third refers to how scientific literacy might enable one “to function minimally in society”[@laugksch2000] (p. 82).
+Second, @laugksch2000 and @norris2014 conceptualise scientific literacy to comprise of three different interpretations of what it means to be ‘literate’. The first refers to what one has learned – the specific knowledge gained. The second refers to being competent, having a certain capacity to engage with scientific contents. The third refers to how scientific literacy might enable one “to function minimally in society” [@laugksch2000, p. 82], that is, being able to the live up to the expectations inherent to a certain role in society, such as being a competent citizen or consumer.
 
-Third, [@roberts2007] provides a useful distinction between two “visions” of scientific literacy, where the term “Vision” is broader than a mere definition and represents an ideal type in the Weberian sense. Vision Iin Roberts’ terms is concerned with literacy or knowledgeability *within science*, that is, it is targeted at an understanding of scientific products (publications, datasets, claims) and processes [@roberts2007] (p. 730). In contrast, Vision II is targeted at situations where scientific knowledge can aid citizens in their daily lives: “At the extreme, this vision can be called *literacy* (again, read *thorough knowledgeability*) *about science-related situations* in which considerations other than science have an important place at the table.” [@roberts2007] (p. 730). As an example, this vision is reflected in how the OECD defines scientific literacy in PISA [@roberts2007] (p. 766):
+Third, @roberts2007 provides a useful distinction between two “Visions” of scientific literacy. Here the term “Vision” is broader than a mere definition and represents an ideal type in the Weberian sense—an intentionally accentuated construct designed to highlight its core features while serving as a conceptual tool for analysis. Vision I in Roberts’ terms is concerned with literacy or knowledgeability within science, that is, it is targeted at an understanding of scientific products (publications, datasets, claims) and processes [@roberts2007, p. 730]. In contrast, Vision II is targeted at situations where scientific knowledge can aid citizens in their daily lives: “At the extreme, this vision can be called literacy (again, read thorough knowledgeability) about science-related situations in which considerations other than science have an important place at the table” [@roberts2007, p. 730]. As an example, this vision is reflected in how the OECD defines scientific literacy in PISA [@roberts2007, p. 766]:
```

Comment:
The italics for "thoroguh knowledgability" where lost. Please check the original version.

Reply:
This is the original: 

```
read *thorough knowledgeability*) *about science-related situations*
```